src/stan-users-guide/efficiency-tuning.qmd

```diff
@@ -1366,102 +1386,83 @@ data {
   vector[N] x;
 }
 transformed data {
+  real mean_x = mean(x);
+  real sd_x = sd(x);
   vector[N] x_std;
-  vector[N] y_std;
-  x_std = (x - mean(x)) / sd(x);
-  y_std = (y - mean(y)) / sd(y);
+  x_std = (x - mean_x) / sd_x;
 }
 parameters {
   real alpha_std;
   real beta_std;
-  real<lower=0> sigma_std;
+  real<lower=0> sigma;
 }
 model {
   alpha_std ~ normal(0, 10);
   beta_std ~ normal(0, 10);
-  sigma_std ~ cauchy(0, 5);
-  for (n in 1:N) {
-    y_std[n] ~ normal(alpha_std + beta_std * x_std[n],
-                      sigma_std);
-  }
+  sigma ~ cauchy(0, 5);
```

Comment:
Use a half-normal here---the Cauchy variables are more trouble than they're worth.

You could add generated quantities here that transform the parameters back to their natural forms (i.e., define `alpha` and `beta`).